Dockerfile

```diff
@@ -0,0 +1,79 @@
+FROM ubuntu:latest
+
+MAINTAINER Maximilian Horzela <maximilian.horzela@kit.edu>
```

Comment:
MAINTAINER is deprecated: https://docs.docker.com/engine/reference/builder/#maintainer-deprecated